astropy/table/table.py

```diff
@@ -247,6 +247,11 @@ def __init__(self, data=None, masked=None, names=None, dtype=None,
 
         default_names = None
 
+        if (isinstance(data, np.ndarray) and
+            data.shape == (0,) and
```

Comment:
Formatting nit to indent the 2nd and 3rd lines here to have a visual offset from the statement block.